src/centralserver/admin-ui/src/views/AppLogin.vue

```diff
@@ -193,10 +193,7 @@ export default (
                 this.loading = false;
               })
               .catch((error) => {
```

Comment:
this catch is for errors from the `then` -pipeline starting after `fetchUserData()`